pagerduty/event_orchestration_path_test.go

```diff
@@ -250,6 +250,55 @@ func TestEventOrchestrationPathGetServiceActiveStatus(t *testing.T) {
 	}
 }
 
+func TestEventOrchestationSanitization(t *testing.T) {
+	setup()
+	defer teardown()
+
+	input := &EventOrchestrationPath{
+		Sets: []*EventOrchestrationPathSet{
+			{
+				ID: "start",
+				Rules: []*EventOrchestrationPathRule{
+					{
+						Actions: &EventOrchestrationPathRuleActions{},
+					},
+				},
+			},
+		},
+	}
+
+	var url = fmt.Sprintf("%s/E-ORC-1/global", eventOrchestrationBaseUrl)
+
+	mux.HandleFunc(url, func(w http.ResponseWriter, r *http.Request) {
+		testMethod(t, r, "PUT")
+		v := new(EventOrchestrationPathPayload)
+		json.NewDecoder(r.Body).Decode(v)
+		if !reflect.DeepEqual(v.OrchestrationPath, input) {
```

Comment:
Test to covert the patch from a regression should be happening at this point, same as You did bellow in `TestEventOrchestrationPathGlobalUpdate`, where You validate the following assertion...

```go
		if !reflect.DeepEqual(string(actionJson), "[]") {
			t.Errorf("empty action array should be []")
		}
```

Because what the test should check is that the input payload is now sanitized and as a result will no longer produce  a **Bad Request** error output from API.

Reply:
This check is already happening in line 294 below, as the `input` that goes into `client.EventOrchestrationPaths.Update` is sanitized. I can move this up if you want though?